sdk/data/aztables/go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/Azure/azure-sdk-for-go/sdk/data/aztables
 
 go 1.18
 
+replace github.com/Azure/azure-sdk-for-go/sdk/azcore => ../../azcore
```

Comment:
Can we release the updated `azcore` today?

Reply:
I don't have an issue with that. Let me do the release and prepare portions.